trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -548,7 +558,8 @@ void TrackerNode::control(
 }
 
 TrackerNode::TrackingResult TrackerNode::getTrackingResult(
-    const tf2::Stamped<tf2::Transform>& robot_to_odom, const Eigen::Vector3d& prediction_offset) const
+    const tf2::Stamped<tf2::Transform>& robot_to_odom, const Eigen::Vector3d& prediction_offset,
+    const double odom_linear_vel, const double odom_rotation_speed) const
```

Comment:
```suggestion
    const double odom_linear_vel, const double odom_angular_vel) const
```
(as the 2nd arg may be negative, it shoud be vel)

Reply:
Done